<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorReport</class>
 <widget class="QDialog" name="ErrorReport">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error Reporter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="thankYouLabel">
     <property name="text">
      <string>Thank you for helping us improve Entomologist by reporting a bug!  Please include your email address.  It will be obscured as an anti-spam measure in our bug tracker.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>What went wrong?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="whatWentWrong"/>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Here is the debug output that will be sent to our server.  Feel free to edit it, or add information:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="debugEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="trackerLabel">
       <property name="text">
        <string>Tracker type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="trackerCombo">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <item>
        <property name="text">
         <string>Unknown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trac</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mantis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bugzilla</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Launchpad</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
